docs/virtual_machines/disks_and_volumes.md

```diff
@@ -674,56 +677,119 @@ the created kubernetes `ConfigMap`.
 Example: Attach the `configMap` to a VM and use `cloudInit` to mount the
 `iso` disk:
 
-    apiVersion: kubevirt.io/v1
-    kind: VirtualMachineInstance
-    metadata:
-      labels:
-        special: vmi-fedora
-      name: vmi-fedora
-    spec:
-      domain:
-        devices:
-          disks:
-          - disk:
-              bus: virtio
-            name: containerdisk
-          - disk:
-              bus: virtio
-            name: cloudinitdisk
-          - disk: {}
-            name: app-config-disk
-            # set serial
-            serial: CVLY623300HK240D
-        machine:
-          type: ""
-        resources:
-          requests:
-            memory: 1024M
-      terminationGracePeriodSeconds: 0
-      volumes:
-      - name: containerdisk
-        containerDisk:
-          image: kubevirt/fedora-cloud-container-disk-demo:latest
-      - cloudInitNoCloud:
-          userData: |-
-            #cloud-config
-            password: fedora
-            chpasswd: { expire: False }
-            bootcmd:
-              # mount the ConfigMap
-              - "mkdir /mnt/app-config"
-              - "mount /dev/$(lsblk --nodeps -no name,serial | grep CVLY623300HK240D | cut -f1 -d' ') /mnt/app-config"
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
         name: cloudinitdisk
-      - configMap:
-          name: app-config
+      - disk: {}
```

Comment:
If disk is the default behaviour you shouldn't need to add a reference here, the VMI mutation webhook should add this for us during submission.

Reply:
Dropped the `disk: {}`  I still need the disk section to set the `serial`.